website/docs/ama/config-file.md

```diff
@@ -14,16 +16,34 @@ The library does not perform any accessibility checks on the production build!
 
 :::
 
-## The log levels
+## Customising the Log Levels
```

Comment:
We are now specifying how to customise the log levels before specifying what they are in the docs, this might be a bit confusing to the reader. Perhaps this should be moved to after the AMA Rules section but before the Constants section.

Reply:
good call!